src/Microsoft.ML.CodeGen/Templates/Console/ModelBuilder.cs

```diff
@@ -390,13 +384,35 @@ public static string GetAbsolutePath(string relativePath)
 public string LabelName {get;set;}
 public bool CacheBeforeTrainer {get;set;}
 public IList<string> PostTrainerTransforms {get;set;}
+public CSharp.CodeGeneratorSettings.GenerateTarget Target {get;set;}
+
+
+void CLI_Annotation()
+{
+this.Write(@"//*****************************************************************************************
+//*                                                                                       *
+//* This is an auto-generated file by Microsoft ML.NET CLI (Command-Line Interface) tool. *
```

Comment:
Nit: wording.  "This file has been auto-generated by" or something like that.

Reply:
@briacht  for you

Reply:
Change to: 
MB: "This file was auto-generated by ML.NET Model Builder."

CLI: "This file was auto-generated by ML.NET CLI (Command-Line Interface) tool."

Reply:
K